pkg/apis/core/validation/validation_test.go

```diff
@@ -8928,223 +8928,6 @@ func TestValidatePodUpdate(t *testing.T) {
 			"Forbidden: pod updates may not change fields other than",
 			"storage resource allocation change",
 		},
-		{
```

Comment:
why were these cases removed?

Reply:
i'm planning to add them to podresourceallocation admission plugin.